### PR TITLE
feat(styles): modify typography desktop style

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/styles/src/lib/typography/desktop.scss [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-// 	font-size: 62.5%;
-// }
+$selectors: (
+	bold: 'b, strong',
+	code: 'pre, var, code, kbd, samp',
+);
 
 :root {
 	@include typography.variables;
 	font-size: 62.5%;
```
Comment (yinonov) [inline libs/styles/src/lib/typography/desktop.scss:15]:
```suggestion
```

**Review by rachelbt — COMMENTED** [review]
the icon in the text-field gets now font-size of 14 and not 16 because nothing is set on it.
All of the snap shor width (and some of the also the height) were change - that strange that it happens - it isn't related to the change you made...

**Comment by yinonov** [comment]
> the icon in the text-field gets now font-size of 14 and not 16 because nothing is set on it. All of the snap shor width (and some of the also the height) were change - that strange that it happens - it isn't related to the change you made...

1. text field's icon size was never set - #655
2. it must have something to do with the change I made here, I ran ui tests in parallel on main

**Comment by yinonov** [comment]
@rachelbt `font-size` and `line-height` both affect block spacing

https://user-images.githubusercontent.com/15140652/188968708-78a2ac71-0d2a-4226-9ade-007d36317e1c.mov

**Review by rachelbt — APPROVED** [review]
went over the snapshots again - some were changed by png width & height as you explained in the screen recording because of plain text (with no vivid typography set on it) inside the component  that was 16px  and now its base - 14px.

**Review by rachelbt — APPROVED** [review]

**Pull request merged**